api/lib/opentelemetry.rb

```diff
@@ -55,6 +66,4 @@ def baggage
   def propagation
     @propagation ||= Context::Propagation::Propagation.new
   end
-
-  self.logger = Logger.new(STDOUT)
```

Comment:
Why not `self.logger = Logger.new(STDOUT, level: ENV['OTEL_LOG_LEVEL'] || Logger::INFO)`?